To be merged 2.8.0 - MTV-2033 RN 2.8.0 (#655)
kubev2v/forklift-documentation: documentation/modules/rn-28-resolved-issues.adoc

In earlier releases of {project-short}, the gateway IP address of the virtual machine was not copied after migration if the IP address was out of network interface subnet range. The issue affected migrations even if you enabled the `Preserve static IPs` setting. This issue has been resolved in {project-short} 2.8.0. link:https://issues.redhat.com/browse/MTV-2018[(MTV-2018)]

.MTV displays an error in the migration plan for virtual machines without a static IP address
In earlier releases of {project-short}, for migration plans in which `Preserve static IPs` is enabled, {project-short} displayed an error message if you did not configure a static IP address for the guest virtual machine. This issue has been resolved in {project-short} 2.8.0. link:https://issues.redhat.com/browse/MTV-1458[(MTV-1458)]

.Disk counter does not update progress while migrating virtual machine with multiple disks
In earlier releases of {project-short}, the `Disk Counter`did not show real-time progress of disk transfer when migrating a virtual machine with multiple disks. This issue has been resolved in {project-short} 2.8.0.link:https://issues.redhat.com/browse/MTV-1718[(MTV-1718)]

> .Incomplete migration plan error for virtual machines with shared disks before cold migration

Author: @RichardHoch @anarnold97 could you review this new resolve issue, please?

// Module included in the following assemblies:
//
// * documentation/doc-Release_notes/master.adoc

[id="rn-28-resolved-issues_{context}"]
= Resolved issues

{project-first} 2.8 has the following resolved issues:

[id="resolved-issues-2-8-0_{context}"]
== Resolved issues 2.8.0

> .Target VMware virtual machines lack connectivity because of unassigned static IP address

Reviewer: ```suggestion
.Target VMware virtual machines lack connectivity because of an unassigned static IP address
```

> In earlier releases of {project-short}, the gateway IP address of the virtual machine was not copied after migration if the IP address was out of network interface subnet range. This issue affected migrations even if you enabled the `Preserve static IPs` setting. This issue has been resolved in {project-short} 2.8.0. link:https://issues.redhat.com/browse/MTV-2018[(MTV-2018)]

Reviewer 2: "was out of network interface subnet range"  Add "the".

Author: Thank you!

> In earlier releases of {project-short}, after migrating Ubuntu 20.04 VMware virtual machines (VM) to {virt} 4.15.6 using {project-short}, the  network interface card (NIC) name of the source VM was not preserved. This resulted in a lack of network connectivity for the VM because the network interface was assigned a new name post migration. The static IP address configured in the network plan before migration was not assigned to the new network interface. This issue has been resolved in {project-short} 2.8.0. link:https://issues.redhat.com/browse/MTV-1607[(MTV-1607)]

Reviewer: ```suggestion
In earlier releases of {project-short}, after migrating Ubuntu 20.04 VMware virtual machines (VM) to {virt} 4.15.6 using {project-short}, the  network interface card (NIC) name of the source VM was not preserved. This resulted in a lack of network connectivity for the VM because the network interface was assigned a new name post-migration. The static IP address configured in the network plan before migration was not assigned to the new network interface. This issue has been resolved in {project-short} 2.8.0. link:https://issues.redhat.com/browse/MTV-1607[(MTV-1607)]
```
i would probably use a compound

> .MTV does not preserve out of range gateway IP address 

Reviewer: ```suggestion
.MTV does not preserve the gateway of a NIC from different subnet
```
i might just simplify this using Martin's [PR](https://github.com/kubev2v/forklift/pull/1378)

> In earlier releases of {project-short}, the gateway IP address of the virtual machine was not copied after migration if the IP address was out of network interface subnet range. The issue affected migrations even if you enabled the `Preserve static IPs` setting. This issue has been resolved in {project-short} 2.8.0. link:https://issues.redhat.com/browse/MTV-2018[(MTV-2018)]

Reviewer: ```suggestion
In earlier releases of {project-short}, the gateway IP address of the virtual machine was not copied after migration if the IP address was out of network interface subnet range. This issue affected migrations even if you enabled the `Preserve static IPs` setting. This issue has been resolved in {project-short} 2.8.0. link:https://issues.redhat.com/browse/MTV-2018[(MTV-2018)]
```

> .MTV displays an error in the migration plan for virtual machines without a static IP address

Reviewer: ```suggestion
.MTV displays an error in the migration plan for VMs without a static IP address
```

> In earlier releases of {project-short}, for migration plans in which `Preserve static IPs` is enabled, {project-short} displayed an error message if you did not configure a static IP address for the guest virtual machine. This issue has been resolved in {project-short} 2.8.0. link:https://issues.redhat.com/browse/MTV-1458[(MTV-1458)]

Reviewer: ```suggestion
In earlier releases of {project-short}, for migration plans in which `Preserve static IPs` is enabled, {project-short} displayed an error message if you did not configure a static IP address for the guest VM. This issue has been resolved in {project-short} 2.8.0. link:https://issues.redhat.com/browse/MTV-1458[(MTV-1458)]
```

> .Disk counter does not update progress while migrating virtual machine with multiple disks

Reviewer: ```suggestion
.Disk counter does not update progress while migrating VMs with multiple disks
```

> In earlier releases of {project-short}, the `Disk Counter` did not show real-time progress of disk transfer when migrating a VM with multiple disks. This issue has been resolved in {project-short} 2.8.0.link:https://issues.redhat.com/browse/MTV-1718[(MTV-1718)]

Reviewer 3: NP: Please add space between the `2.8.0.` and link

> In earlier releases of {project-short}, the `Disk Counter`did not show real-time progress of disk transfer when migrating a virtual machine with multiple disks. This issue has been resolved in {project-short} 2.8.0.link:https://issues.redhat.com/browse/MTV-1718[(MTV-1718)]

Reviewer 4: ```suggestion
In earlier releases of {project-short}, the `Disk Counter` did not show real-time progress of disk transfer when migrating a virtual machine with multiple disks. This issue has been resolved in {project-short} 2.8.0.link:https://issues.redhat.com/browse/MTV-1718[(MTV-1718)]
```

> In earlier releases of {project-short}, the `Disk Counter` did not show real-time progress of disk transfer when migrating a virtual machine with multiple disks. This issue has been resolved in {project-short} 2.8.0.link:https://issues.redhat.com/browse/MTV-1718[(MTV-1718)]

Reviewer: ```suggestion
In earlier releases of {project-short}, the `Disk Counter` did not show real-time progress of disk transfer when migrating a VM with multiple disks. This issue has been resolved in {project-short} 2.8.0.link:https://issues.redhat.com/browse/MTV-1718[(MTV-1718)]
```

> .Incomplete migration plan error for virtual machines with shared disks before cold migration

Reviewer: ```suggestion
.Incomplete migration plan error for VMs with shared disks before cold migration
```

> In earlier releases of {project-short}, the migration plan for a virtual machine (VM) with shared disks showed an error that did not list all VMs to be powered off before the cold migration. This issue has been resolved in {project-short} 2.8.0. link:https://issues.redhat.com/browse/MTV-2206[(MTV-2206)]

Reviewer: ```suggestion
In earlier releases of {project-short}, the migration plan for a VM with shared disks showed an error that did not list all VMs to be powered off before the cold migration. This issue has been resolved in {project-short} 2.8.0. link:https://issues.redhat.com/browse/MTV-2206[(MTV-2206)]
```

.Static IP address is not preserved for VMs with manual MAC address configuration

> In earlier releases of {project-short}, the static IP address of a VMware VM was not preserved if you configured a MAC address with upper case for the virtual network adapter of the VM. This issue affected migrations where the `Preserve static IPs` setting was enabled and has been resolved in {project-short} 2.8.0. link:https://issues.redhat.com/browse/MTV-2285[(MTV-2285)] 

Reviewer: ```suggestion
In earlier releases of {project-short}, the static IP address of a VMware VM was not preserved if you configured a MAC address with uppercase for the virtual network adapter of the VM. This issue affected migrations where the `Preserve static IPs` setting was enabled and has been resolved in {project-short} 2.8.0. link:https://issues.redhat.com/browse/MTV-2285[(MTV-2285)] 
```

Author: Thank you! Corrected it.
